web/controllers/skill_controller.ex

```diff
@@ -23,7 +23,7 @@ defmodule CodeCorps.SkillController do
     |> limit_filter(params)
   end
 
-  def search(_conn, params) do
+  def search(_conn, %{"query" => query} = params) do
```

Comment:
You can remove the ` = params` portion here if you're not using the `params` as a whole object.